testsuite/tests/kuadrant/authorino/operator/clusterwide/test_wildcard_collision.py

```diff
@@ -0,0 +1,44 @@
+"""
+Test for wildcard collisions with clusterwide authorino
+"""
+
+import pytest
+
+from testsuite.openshift.objects.auth_config import AuthConfig
+
+
+# pylint: disable = unused-argument
+@pytest.fixture(scope="module")
+def authorization(authorino, blame, openshift, module_label, envoy):
+    """In case of Authorino, AuthConfig used for authorization"""
+    return AuthConfig.create_instance(openshift, blame("ac"), "*.redhat.com", labels={"testRun": module_label},
+                                      response={"name": "header",
+                                                "json": {"properties": [{"name": "anything", "value": "one"}]}})
+
+
+# pylint: disable = unused-argument
+@pytest.fixture(scope="module")
+def authorization2(authorino, blame, openshift2, module_label, envoy):
+    """In case of Authorino, AuthConfig used for authorization"""
+    return AuthConfig.create_instance(openshift2, blame("ac"), "*.redhat.com", labels={"testRun": module_label},
+                                      response={"name": "header",
+                                                "json": {"properties": [{"name": "anything", "value": "two"}]}})
+
+
+@pytest.mark.parametrize("client_fixture", [
+    pytest.param("client", id="First namespace"),
+    pytest.param("client2", id="Second namespace"),
+])
+def test_wildcard_collision(client_fixture, request):
+    """
+    Preparation:
+        - Create AuthConfig with host set to `*.redhat.com`
+        - Create AuthConfig with host set to `*.redhat.com` in another project
+    Test:
+        - Send request to authorino
+        - Assert that the correct AuthConfig was used
+    """
+    client = request.getfixturevalue(client_fixture)
+    response = client.get("/get")
+    assert response.status_code == 200
+    assert response.json()["headers"]["Header"] == '{"anything":"one"}'
```

Comment:
Could you also check that it the host of the second AuthConfig is rejected? It should be referenced in the status of the AuthConfig.

```
status:
  summary:
    hostsReady:
      - hosta
```